modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/MetaStorageManager.java

```diff
@@ -174,7 +174,7 @@ public MetaStorageManager(
 //        );
 
         // TODO: IGNITE-14414 Cluster initialization flow. Here we should complete metaStorageServiceFuture.
-        clusterNetSvc.messagingService().addMessageHandler((message, senderAddr, correlationId) -> {});
+//        clusterNetSvc.messagingService().addMessageHandler((message, senderAddr, correlationId) -> {});
```

Comment:
What is this exactly? Can we simply remove this line?

Reply:
I don't know =( But there's a TODO, so its author might need it...

Reply:
Ok